torch/jit/__init__.py

```diff
@@ -1469,6 +1469,17 @@ def __init__(self, original, stubs):
             # Copy constants
             self.__dict__["_constants_set"] = set(getattr(original, "__constants__", []))
 
+            # Automatically register primitive types as attributes
+            for name in dir(original):
```

Comment:
It seems strange that we are taking all values in dir(original) - won't be we be getting arbitrary python implementation details here ?